Source/ablastr/utils/SignalHandling.H

```diff
@@ -0,0 +1,85 @@
+/* Copyright 2022 Philip Miller
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#ifndef ABLASTR_SIGNAL_HANDLING_H_
+#define ABLASTR_SIGNAL_HANDLING_H_
+
+#include <AMReX_Config.H>
+
+#if defined(AMREX_USE_MPI)
+#include <mpi.h>
+#endif
+
+#include <atomic>
+#include <string>
+
+/**
```

Comment:
In ABLASTR we started to not let the namespacing slip :)
```suggestion

namespace ablastr::utils {
/**
```